tecken/useradmin/management/commands/superuser.py

```diff
@@ -31,12 +31,16 @@ def handle(self, *args, **options):
                 username = default_username_algo(email)
             user = User.objects.create(username=username, email=email)
             user.set_unusable_password()
-            self.stdout.write(self.style.WARNING("User created"))
+            self.stdout.write(self.style.WARNING("New user created"))
 
-        user.is_superuser = not user.is_superuser
```

Comment:
This used to enable/disable superuser status, but we don't need that. In a local dev environment, we need a command that creates a user account and assigns superuser/staff to it. That simplifies a ton of things when developing/testing.